libraries/provision/ansible/playbooks/tasks/create-sync-gateway-user.yml

```diff
@@ -1,8 +1,3 @@
-# Add sync_gateway user on Centos/linux/ubuntu/debian
```

Comment:
This passes for the sanity tests I assume, but are there other tests that need a SG user that no longer pass without this?

Reply:
The sync_gateway user is created by default in the installation, so I think we don't need it. The problem is that the old Ansible version is not compatible with Debian12, causing this user creation to fail.